spyder/widgets/comboboxes.py

```diff
@@ -264,8 +264,11 @@ def focusOutEvent(self, event):
     def _complete_options(self):
         """Find available completion options."""
         text = to_text_string(self.currentText())
-        opts = glob.glob(text + "*")
-        opts = sorted([opt for opt in opts if osp.isdir(opt)])
+        glob_opts = glob.glob(text + "*")
+        opts = sorted(
+            [os.path.normcase(opt) for opt in glob_opts if osp.isdir(opt)])
+        opts += sorted(
+            [os.path.normcase(opt) for opt in glob_opts if opt not in opts])
```

Comment:
It seems this is necessary to provide file completions in the working directory toolbar (at least that's what I found while testing this PR).

In that case, I disagree with these changes due to the following reasons:

* The working directory toolbar is used to switch directories, not to also select files. I agree with your reasoning of allowing users to open files through it when pasting their names, because that follows an intuitive expectation. But this functionality is different and, from a UX point of view, it'd be confusing for regular users to see directories and files listed on it.
* This could have a performance impact if the inspected directory has 1000 or 10000 files. I'm worried that in that case the interface will freeze trying to provide all those completions, which is something we shouldn't allow.
* You can always open files from the IPython console using the `%edit` magic (as I mentioned before), so I don't think this is really necessary. Furthermore, since directory inspection runs in the kernel, that doesn't have the chance to block the interface.

Reply:
I removed it